mlflow/tensorflow.py

```diff
@@ -0,0 +1,91 @@
+"""MLflow integration for Tensorflow."""
+
+from __future__ import absolute_import
+
+import os
+import dill as pickle
+
+import pandas
+import tensorflow as tf
+
+from mlflow.utils.file_utils import TempDir
+from mlflow import pyfunc
+from mlflow.models import Model
+import mlflow.tracking
+
+
+class TFWrapper(object):
+    """Wrapper class that creates a predict function such that 
+    predict(data: pandas.DataFrame) -> pandas.DataFrame"""
+    def __init__(self, model_fn, model_dir):
+        self.estimator = tf.estimator.Estimator(model_fn, model_dir=model_dir)
+
+    def predict(self, df):
+        input_fn = tf.estimator.inputs.pandas_input_fn(df, shuffle=False)
+        pred = self.estimator.predict(input_fn)
+        results = []
+        for p in pred:
+            results.append(p['predictions'])
+        return pandas.DataFrame(results)
+
+
+def save_model(tf_model, path, conda_env=None, mlflow_model=Model()):
+    """Save a Tensorflow model to a directory in the local file system."""
+    if os.path.exists(path):
+        raise Exception("Path '{}' already exists".format(path))
+    os.makedirs(path)
+    model_fn_file = os.path.join(path, "model_fn.pkl")
+    model_dir_file = os.path.join(path, "model_dir.pkl")
```

Comment:
`model_dir` is a string (path to the estimator's model directory) right? It'd be good to directly copy the model directory into the directory at `path`. That way you could share your exported model directory with me & I could load the model back (I'd otherwise end up trying to read from a directory on your local filesystem).